arangodb-net-standard/IndexApi/IIndexApiClient.cs

```diff
@@ -32,10 +32,9 @@ internal interface IIndexApiClient
         /// <summary>
         /// Creates a new index
         /// </summary>
-        /// <param name="indexType">The type of index to create.</param>
         /// <param name="query">Query parameters for the request.</param>
         /// <param name="body">The properties of the new index.</param>
         /// <returns></returns>
-        Task<IndexResponseBase> PostIndexAsync(IndexType indexType, PostIndexQuery query, PostIndexBody body);
+        Task<IndexResponseBase> PostIndexAsync(PostIndexQuery query, PostIndexBody body);
```

Comment:
We could also provide convenience methods for specific index types, so that callers have access to a simpler set of methods instead of having to create objects and set the correct properties. For example:

```
Task<IndexResponseBase> PostPersistentIndexAsync(string collectionName, string indexType/*, other relevant optional parameters here, e.g. bool estimates*/);
```

What do you think? If it makes sense to you I can open a separate pull request to do that.

Reply:
Yes it makes sense, but preferably under a separate PR?

Reply:
Agreed, I will create an issue once the pull request is merged to track this enhancement.